specification/devcenter/DevCenter/DevCenter/routes.tsp

```diff
@@ -14,4 +15,32 @@ interface DevCenter {
 
   @doc("Gets a project.")
   getProject is StandardResourceOperations.ResourceRead<Project>;
+
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "no @key for ProjectAbility model"
```

Comment:
Are you saying that ProjectAbility is not a resource?

Reply:
This is a weird one, it's more of a LIST route that returns two arrays containing a set of permissions/role assignments that the user has assigned to them over the project (the allowed values of these arrays are both enums). So there's no key here. 

I wonder if perhaps the wording and naming should be changed to listProjectAbilities instead? The person who wrote this API is unfortunately OOF for a few days so I can't ask them, but let me know if you have any thoughts.